src/llvm_ir_generator.cpp

```diff
@@ -261,11 +267,26 @@ void LLVMIRGenerator::Visit(const RecordVarDeclNode& record_var_decl) {
   for (auto i = std::size_t{0}, e = record_var_decl.inits.size(),
             slot_count = record_type.SlotCount();
        i < slot_count && i < e; ++i) {
+    // TODO: consider the case for union when there are multiple designators. It
+    // seems that both Clang and gcc have initialized the following union with
+    // the value 6.
```

Comment:
I think we can directly say that the last one is used to initialized.
```suggestion
    // seems that both Clang and gcc initialized with the last one.
    // For example, following union is initialized with the value 6.
```